query/reverse_defaults.js

```diff
@@ -1,10 +1,10 @@
 
-var peliasQuery = require('pelias-query'),
-    extend = require('extend');
+var peliasQuery = require('pelias-query');
+var _ = require('lodash');
 
-module.exports = extend( false, peliasQuery.defaults, {
+module.exports = _.merge({}, peliasQuery.defaults, {
```

Comment:
BUG FIX: `extend` will assign the resulting object to `peliasQuery.defaults`, thereby poisoning the clean well of default data for all subsequent uses of `peliasQuery.defaults`. See same fix in all 3 default files.